examples/prisma-example/apps/api/src/resources/account/actions/forgot-password.ts

```diff
@@ -0,0 +1,59 @@
+import { userService } from 'resources/user';
+
+import { validateMiddleware } from 'middlewares';
+import { emailService } from 'services';
+import { securityUtil } from 'utils';
+
+import config from 'config';
+
+import { forgotPasswordSchema } from 'schemas';
+import { AppKoaContext, AppRouter, ForgotPasswordParams, Next, Template, User } from 'types';
+
+interface ValidatedData extends ForgotPasswordParams {
+  user: User;
+}
+
+async function validator(ctx: AppKoaContext<ValidatedData>, next: Next) {
+  const user = await userService.findUnique({ where: {email: ctx.validatedData.email} });
```

Comment:
```suggestion
  const user = await userService.findUnique({ where: { email: ctx.validatedData.email } });
```

enable prettier & eslint